package.json

```diff
@@ -32,6 +32,7 @@
     "detective": "^4.2.0",
     "detective-es6": "1.1.2",
     "graphviz": "0.0.8",
+    "precinct": "^2.5.1",
```

Comment:
Precinct only [uses a caret for detective-es6](https://github.com/mrjoelkemp/node-precinct/blob/master/package.json#L31) (effectively upgrading 1.1.3), [sniffs the module type](https://github.com/mrjoelkemp/node-precinct/blob/master/index.js#L45), and then [delegates to detective-es6](https://github.com/mrjoelkemp/node-precinct/blob/master/index.js#L57). 

Try running `npm install --save detective-es6@1.1.3` in your madge fork and seeing if that works.